src/tagify.js

```diff
@@ -489,8 +489,10 @@ Tagify.prototype = {
                         e.target.textContent = e.target.originalValue;
 
                     case 'Enter' :
+                    case 'Tab' :
                         e.preventDefault();
                         e.target.blur();
+                        this.trigger("edit", e.target.textContent);
```

Comment:
Thanks for the PR but the event being triggered hasn't been placed currently.
"enter" & "tab" cases only exist so they would trigger the `blur` event.

Where the event should actually be triggered is within the method `onEditTagBlur`,
so move `this.trigger("edit", e.target.textContent);` to the end of that method.

Reply:
@yairEO, done that.